InventoryCatalog/Model/ResourceModel/StockStatusFilter.php

```diff
@@ -53,6 +55,10 @@ public function execute(
             []
         );
         $select->where("{$stockStatusTableAlias}.{$isSalableFieldName} = ?", 1);
```

Comment:
row 55 should be removed, check https://github.com/magento-commerce/inventory/pull/191/files#diff-1992bd5ddaf91593cb99e8f27dcf6fda8343f5ff3cd0e0f86f564601ea7f7ed1L55